blogpost1.php

```diff
@@ -1,34 +1,7 @@
-<html>
-  <head>
-    <title> Contact </title>
-    <link rel="stylesheet" href="style.css?Time=<?php echo microtime()?>" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-  </head>
-  <body>
-      <h1>
-          More Info About Me </h1>
-          <div class = "active">
-                <ul>
-                <li>
-                    <a href = "index.php">
-                    Home
-                </a> </li>
-                    <li>
-                        <a href = "blogpost1.php">
-                            Contact
-                    </a> </li>
-
-                    <li>
-                        <a href = "Resume.php">
-                         Projects/Resume </a> </li>
-                  <li>
-                      <a href = "blogpost2.php">
-                          Summer Blog </a> </li>
-                </ul>
-          </div>
-
-     <br>
-     <div style='max-width: 90%; background-color: #B3C3E4; padding:10px; margin:auto; border-radius:10px'>
+<?php
+include('config/init.php');
```

Comment:
I realize you're still working on this stuff, and maybe this file doesn't serve a purpose, but is the plan to move this into the `view_post.php` functionality? Or is this a test file that I should be ignoring?